src/pages/productPage.ts

```diff
@@ -103,4 +111,14 @@ export default class ProductPage extends BasePage {
       await expect(this.productPageLocators.productCountInCompareListText).toHaveText(expectedRegex, { timeout: 15000 });
     });
   }
+
+  async verifyProductInWishlistByName(productName: string) {
+    await test.step(`Verify product "${productName}" is added to Wishlist`, async () => {
+      // Локатор для повідомлення про успішне додавання до списку бажаного
```

Comment:
Try to write all your code and comments using English. It's make your code more readable and friendly for other developers.
Also it's a good point to improve your English skills.